go.mod

```diff
@@ -7,15 +7,15 @@ require (
 	github.com/giantswarm/microclient v0.2.0
 	github.com/giantswarm/microendpoint v0.2.0
 	github.com/giantswarm/microerror v0.2.0
-	github.com/giantswarm/microkit v0.2.0
+	github.com/giantswarm/microkit v0.2.1
 	github.com/giantswarm/micrologger v0.3.1
-	github.com/giantswarm/operatorkit v0.2.0
+	github.com/giantswarm/operatorkit v1.2.0
 	github.com/go-kit/kit v0.10.0
-	github.com/gorilla/mux v1.7.3
-	github.com/prometheus/client_golang v1.3.0
-	github.com/spf13/viper v1.6.2
+	github.com/gorilla/mux v1.7.4
+	github.com/prometheus/client_golang v1.7.1
+	github.com/spf13/viper v1.7.0
 	gopkg.in/resty.v1 v1.12.0
-	k8s.io/api v0.16.6
-	k8s.io/apimachinery v0.16.6
-	k8s.io/client-go v0.16.6
+	k8s.io/api v0.17.2
```

Comment:
@xh3b4sd would it be ok to update it?
currently we block all k8s.io dependencies which are >=0.17.0

Reply:
Certain combinations may work but we should rather stick to the permutation we are using in other projects. Otherwise we have too many snowflakes which makes maintenance hard.

Reply:
I would prefer to stick to 0.16.6 for these and upgrade all of our components at once. We would need to find out if these are transitive dependencies from another upgraded library or if they can simply be reverted.